app/scripts/metamask-controller.test.js

```diff
@@ -2025,6 +2030,73 @@ describe('MetaMaskController', () => {
         expect(TokenListController.prototype.start).toHaveBeenCalledTimes(1);
       });
     });
+
+    describe('start/stop rates controller', () => {
+      const mockEVMAccount = createMockInternalAccount();
+      const mockNonEvmAccount = {
+        ...mockEVMAccount,
+        type: BtcAccountType.P2wpkh,
+        methods: [BtcMethod.SendMany],
+        address: 'bc1qar0srrr7xfkvy5l643lydnw9re59gtzzwf5mdq',
+      };
+
+      beforeEach(() => {
+        jest.spyOn(metamaskController.ratesController, 'start');
+        jest.spyOn(metamaskController.ratesController, 'stop');
+      });
+
+      afterEach(() => {
+        jest.clearAllMocks();
+      });
+
+      it('starts rates controller if selected account is non-EVM', async () => {
+        expect(metamaskController.ratesController.start).not.toHaveBeenCalled();
+
+        metamaskController.controllerMessenger.publish(
+          'AccountsController:selectedAccountChange',
+          mockNonEvmAccount,
+        );
+
+        expect(metamaskController.ratesController.start).toHaveBeenCalledTimes(
+          1,
+        );
+      });
+
+      it('stops rates controller if selected account is non-EVM', async () => {
+        expect(metamaskController.ratesController.start).not.toHaveBeenCalled();
+
+        metamaskController.controllerMessenger.publish(
+          'AccountsController:selectedAccountChange',
+          mockNonEvmAccount,
+        );
+
+        expect(metamaskController.ratesController.start).toHaveBeenCalledTimes(
+          1,
+        );
+
+        metamaskController.controllerMessenger.publish(
+          'AccountsController:selectedAccountChange',
+          mockEVMAccount,
+        );
+        expect(metamaskController.ratesController.start).toHaveBeenCalledTimes(
+          1,
+        );
+        expect(metamaskController.ratesController.stop).toHaveBeenCalledTimes(
+          1,
+        );
+      });
+
+      it('does not start rates controller if selected account is non-EVM', async () => {
```

Comment:
```suggestion
      it('does not start rates controller if selected account is EVM', async () => {
```
Same comment as my above suggestion